src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -366,6 +374,10 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
     result_code = mobile_apis::Result::WRONG_LANGUAGE;
   }
 
+  if (application_manager_.hmi_interfaces().GetInterfaceState(
```

Comment:
@pestOO  No, it han be in 3 different stated:
Avaliable
Not avaliable
NO_RESPONSE